test/common/tmpdir.js

```diff
@@ -5,17 +5,27 @@ const fs = require('fs');
 const path = require('path');
 const { pathToFileURL } = require('url');
 const { isMainThread } = require('worker_threads');
+const isUnix = process.platform === 'darwin' || process.platform === 'linux';
```

Comment:
I'm not sure if this is the logic we want. Would `process.platform !== 'win32'` be more appropriate?

Reply:
Also, `isPosix` or `isUnixLike` would be a better name for the variable

Reply:
I'm fine with changing this, if we are all Ok with the change?